src/default.js

```diff
@@ -117,6 +129,10 @@
       }
     };
 
+    var isFunction = function (func) {
+      return typeof func === "function";
```

Comment:
Because this code should be as short as possible we don't need the 3 `=`'s here. I'm a bit anal about the public script, but I think it's important to not add extra bytes. That's why we added all those variables at the top of this script.

```suggestion
      return typeof func == "function";
```